### DISCUSSION TITLE
请支持 Azure OpenAI Service 翻译，类似 GPT3.5-turbo，只是接口地址不同

### DISCUSSION THREAD

**Discussion by hh23485** [immersive-translate/immersive-translate#490] [Ideas]
### 新功能 | New Feature

1. 除了 OpenAI 之外，希望支持 GPT3.5-turbo

### 为什么您想要添加这些功能 | Why you want to add these features

没有 openai 账户，但有 Azure 账户，且在国内更容易访问

### 提议 | Proposal

修改 openai 的版本来对 azure openai 进行支持
可以贡献

**Comment by theowenyoung** [comment]
已支持的话～ 直接填写自定义网址和 key就可以了。

**Comment by hh23485** [comment]
@theowenyoung , 然而接口和 OpenAI 并不完全一样，并不能使用相同的实现，我没有详细比较，但试了一下自定义网址和key是不行的，需要额外支持，只是模型相同。

**Comment by theowenyoung** [comment]
自定义接口要写全部～这里有示例：

https://immersive-translate.owenyoung.com/services/openai#azure-openai
